cms/envs/aws.py

```diff
@@ -132,6 +132,9 @@
 
 LMS_BASE = ENV_TOKENS.get('LMS_BASE')
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
+LMS_INTERNAL_ROOT_URL = ENV_TOKENS.get('LMS_INTERNAL_ROOT_URL', LMS_ROOT_URL)
```

Comment:
@fredsmith Is `LMS_INTERNAL_ROOT_URL` overridden appropriately somewhere for prod-edx-edxapp? I couldn't find it anywhere.

Reply:
No, there will need to be a corresponding configuration PR to be able to configure this variable.

Reply:
@douglashall I'm not sure what prod-edx-edxapp is. edx/configuration PR to configure the `LMS_INTERNAL_ROOT_URL` variable is at https://github.com/edx/configuration/pull/4200. Is there anything else that I need to do?